Update README.md (#14502)
goharbor/harbor: README.md

* **Developer Group:** Join Harbor developer group: [harbor-dev@lists.cncf.io](https://lists.cncf.io/g/harbor-dev) for discussion on Harbor development and contribution.
* **Slack:** Join Harbor's community for discussion and ask questions: [Cloud Native Computing Foundation](https://slack.cncf.io/), channel: [#harbor](https://cloud-native.slack.com/messages/harbor/) and [#harbor-dev](https://cloud-native.slack.com/messages/harbor-dev/)

> ## Community contributed CLI

Reviewer: ```suggestion
## Additional Tools
Tools layered on top of Harbor and contributed by the community.
```

### Harbor CLI

> A community contributed [Unofficial Harbor CLI](https://github.com/hinyinlam-pivotal/harbor-cli) can be downloaded to interact with Harbor in system level and repo level.

Reviewer: ```suggestion
* [Unofficial Harbor CLI](https://github.com/hinyinlam-pivotal/harbor-cli)
  - Interact with Harbor in system level and repo level. The `harbor-cli` is built on top of `harbor-client-java`, tutorial and usage examples also included in the author's repo.
  - Contributed by [hinyinlam-pivotal](https://github.com/hinyinlam-pivotal). For any feedback, improvement and issues in Java SDK and CLI, please contact the author.
```

> The `harbor-cli` is built on top of `harbor-client-java`, tutorial and usage examples also included in the author's repo.

Reviewer: ```suggestion

```

> For any feedback, improvement and issues in Java SDK and CLI, please contact the [author](https://github.com/hinyinlam-pivotal).

Reviewer: ```suggestion

```

Merge together.
